src/context/Provider.js

```diff
@@ -4,12 +4,26 @@ import PropTypes from 'prop-types';
 import initialState from './initialState';
 import globalReducer from './reducer';
 
+const getCountry = async () => {
+  return fetch('http://ip-api.com/json')
```

Comment:
if its not too much work maybe leveraging this browser built in geolocation is better style? idk
https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition
we can consider for next time

Reply:
Right, I look into it and it seems like it gives the latitude and longitude so we might also need to find a way to know the countries boundaries, will consider it for next time:')